src/apiv2.ts

```diff
@@ -4,16 +4,17 @@ import { parse, URLSearchParams } from "url";
 import * as ProxyAgent from "proxy-agent";
 import AbortController from "abort-controller";
 import fetch, { HeadersInit, Response, RequestInit, Headers } from "node-fetch";
+import * as auth from "./auth";
 
 import { FirebaseError } from "./error";
 import * as logger from "./logger";
 import * as responseToError from "./responseToError";
 
-const CLI_VERSION = require("../package.json").version;
+import CLI_VERSION from "../package.json";
```

Comment:
I find it unlikely that this variable would remain the same. Would the location using this need to change to do something like `CLI_VERSION.version`? in which case, should this have a better name?

Reply:
Wow. you guys are quick on the draw. I didn't intend this to be reviewed yet. The correct version is `import {version as CLI_VERSION} from '../package.json'`

Reply:
OIC. The git workflow automatically added people as reviewers? I meant this to be a draft until now.

Reply:
Nope I just saw it in my inbox and was curious!  I was not added as a reviewer.  Since the CLI has no real staffing or maintainers for common bits like this you'll get a lot of drive-by gawkers.

Carry on!

Reply:
It doesn't automatically recommend folks - but I think the button to create a PR defaults to whatever you did last? Either way, we happened to see it :P Sorry we're super eager!

Reply:
Hi Sam! bahaha